config/endpoints/api_particulier/cnaf_msa_quotient_familial_v2.yml

```diff
@@ -1,5 +1,7 @@
 ---
 - uid: 'cnav/quotient_familial_v2'
+  opening: protected
```

Comment:
Idem ici en vrai flip it non ?

Reply:
c'est à changer par dorine, c'est des placeholders

Reply:
Pour API Particulier tout est en protected mais c'est pas une mauvaise idée de le rappeler dans le tableau du cas d'usage. Par contre, c'est peut-être pas nécessaire d'avoir à le remplir à chaque fois, mais ça implique de modifier le code